lib/system/dollars.nim

```diff
@@ -75,33 +75,28 @@ proc `$`*[T: tuple|object](x: T): string =
   ##   $(a: 23, b: 45) == "(a: 23, b: 45)"
   ##   $() == "()"
   result = "("
-  var firstElement = true
-  const isNamed = T is object or isNamedTuple(T)
-  when not isNamed:
-    var count = 0
+  let isNamed = isNamedTuple(T)
```

Comment:
Can be const like in the original no?

Reply:
Can be const for sure. But that wouldn't change anything. I prefer `let` here as it is there more common way to declare a variable and therefore also smoother to read. And that is what I am optimizing for, readability.

Reply:
```suggestion
  const isNamed = isNamedTuple(T)
```
there you go :p